packages/dmn-js-shared/src/base/Manager.js

```diff
@@ -109,15 +109,20 @@ export default class Manager {
       }
 
       if (err || !options.open) {
-        this._emit('import.done', { error: err, warmings: parseWarnings });
+        this._emit('import.done', { error: err, warnings: parseWarnings });
 
         return done(err, parseWarnings);
       }
 
       var view = this._activeView || this._getInitialView(this._views);
 
       if (!view) {
-        return done(new Error('no displayable contents'));
+        var noDisplayableContentsErr = new Error('no displayable contents');
```

Comment:
Great catch.

I believe I found a similar inconsistency in bpmn-js as we introduced our modern import :smile:.

As `var` in ES5 gets hoisted to the outer scope my proposal would be to simply override the existing `err` variable. We should avoid declaring a nested `var`.

Reply:
Thanks, good point. I corrected that.